### DISCUSSION TITLE
Add the Stepper component

### DISCUSSION THREAD

**Discussion by joshgoldenbergno10** [mantinedev/mantine#2948] [Feature requests]
https://mantine.dev/core/stepper/

As discussed on discord - we want the ability to have a high level of timeline of the things that need to happen and then when you click on each step you see eg visualisations or information about the decisions. We currently use tabs quite a lot for things adjacent to this but the problem with them is it's pretty hard to give extra descriptive information on them which is viewable across all tabs without making it look quite ugly (eg putting content above it and fiddling a lot with text and sizes.

**Comment by rtivital** [comment]
I do not get the issue, we already provide stepper and tabs components

**Reply by joshgoldenbergno10** [reply]
apologies, was in the wrong repo, meant dash-mantine-components! feel free to close
